### PR TITLE
[KIECLOUD-56] - Update PAM DM templates on OpenShift for 7.1.1

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

File: official/fis/imagestreams/apicurito-ui-rhel7.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "tags": [
+            {
+                "annotations": {
+                    "description": "Red Hat Apicurito UI image.",
+                    "openshift.io/display-name": "Red Hat Apicurito UI",
+                    "tags": "apicurio,hidden"
+                },
+                "from": {
+                    "kind": "DockerImage",
+                    "name": "apicurio/apicurito-ui:latest"
```
Comment (bparees) [inline official/fis/imagestreams/apicurito-ui-rhel7.json]: please fully qualify all docker image references w/ the registry name.

Comment (bparees) [reply official/fis/imagestreams/apicurito-ui-rhel7.json]: also this should be using the local reference policy if the image is hosted on registry.redhat.io.  (see other imagestreamtags for an example)

File: official/fis/imagestreams/fuse-apicurito-generator-rhel7.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        "tags": [
+            {
+                "annotations": {
+                    "description": "Red Hat Apicurito Generator image.",
+                    "openshift.io/display-name": "Red Hat Apicurito Generator",
+                    "tags": "apicurio,fuse,hidden"
+                },
+                "from": {
+                    "kind": "DockerImage",
+                    "name": "apicurio/fuse-apicurito-generator:latest"
```
Comment (bparees) [inline official/fis/imagestreams/fuse-apicurito-generator-rhel7.json]: fully qualify + add local ref policy

Comment (spolti) [reply official/fis/imagestreams/fuse-apicurito-generator-rhel7.json]: Hi @bparees I executed the import_content.py script and pushed all the changes from its result, is it the correct way right? I am not responsible for this fuse imagestreams. The content I wish to update is rhpam and rhdm.

Comment (bparees) [reply official/fis/imagestreams/fuse-apicurito-generator-rhel7.json]: ok.  it sounds like whoever owns the fuse imagestreams has done this incorrectly, not your problem.

**Pull request merged**